javascript-forms/form-events.js

```diff
@@ -0,0 +1,30 @@
+function handleFocus(event) {
+  console.log('The focus event was fired.');
+  console.log(event.target.name);
+}
+function handleBlur(event) {
+  console.log('The blur event was fired.');
+  console.log(event.target.name);
+}
+
+function handleInput(event) {
+  console.log('The input event was fired.');
+  console.log(event.target.name);
+  console.log(event.target.value);
+}
+
+var $name = document.forms[0].name;
+var $email = document.forms[0].email;
+var $message = document.forms[0].message;
```

Comment:
Selecting a `<form>` by its index is dependent on the HTML's source order. Try using a selection method that isn't dependent on where the form is.